scripts/release.sh

```diff
@@ -62,7 +62,14 @@ main() {
       BRANCH=$(git rev-parse --abbrev-ref HEAD)
   else
       REPOSITORY=${REPOSITORY:-"git@github.com:etcd-io/etcd.git"}
-      BRANCH=${BRANCH:-"release-${MINOR_VERSION}"}
+      if [ "${MINOR_VERSION}" == "3.6" ]; then
```

Comment:
For non-released branches I expect that script will expect caller will be picking the branch manually.

Reply:
That sounds good; let's go with updating the documentation, then.